### PR TITLE
feat(NotificationLycTab) :#MLYSTOR-598 NotificationLycTab export chan…

### PR DESCRIPTION
**Pull request by Rseuret**
## Describe your changes
NotificationLycTab export changes ( colors for label)
## Checklist tests
Les subventions sont en jaune dans l export et le reste en vert
## Issue ticket number and link
https://jira.support-ent.fr/browse/MLYSTOR-598

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [x] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: src/main/java/fr/openent/lystore/export/validOrders/listLycee/ListLycee.java [hunk 1]
```diff
@@ -35,17 +34,17 @@ protected void initDatas(Handler<Either<String, Boolean>> handler) {
 
     private void writeArray(Handler<Either<String, Boolean>> handler) {
 
-        excel.insertWithStyle(0,0,"Numéro de validation",excel.labelOnYellow);
-        excel.insertWithStyle(1,0,"Numéro du marché",excel.labelOnYellow);
-        excel.insertWithStyle(2,0,"Libellé du marché",excel.labelOnYellow);
-        excel.insertWithStyle(3,0,"Date du BC",excel.labelOnYellow);
+        excel.insertWithStyle(0,0,"Numéro de validation",excel.labelOnLightYellow);
```
Comment (Rseuret) [inline src/main/java/fr/openent/lystore/export/validOrders/listLycee/ListLycee.java:37]: refacto

File: src/main/java/fr/openent/lystore/export/validOrders/listLycee/ListLycWithPrice.java [hunk 2]
```diff
@@ -249,25 +249,25 @@ private int inserTotal(int initx, int currentI) {
...
-        excel.insertWithStyle(10,1,"DATE ARC",excel.labelOnYellow);
-        excel.insertWithStyle(11,1,"DATE LIMITE -LIVRAISON",excel.labelOnYellow);
-        excel.insertWithStyle(12,1,"DATE CSF",excel.labelOnYellow);
-        excel.insertWithStyle(13,1,"NUMERO FACTURE ",excel.labelOnYellow);
-        excel.insertWithStyle(14,1,"DATE FACTURE",excel.labelOnYellow);
-        excel.insertWithStyle(15,1,"MONTANT FACTURE",excel.labelOnYellow);
-        excel.insertWithStyle(16,1,"FACTURE / RECAP ",excel.labelOnYellow);
-        excel.insertWithStyle(17,1,"RETARD DE LIVRAISON ",excel.labelOnYellow);
-        excel.insertWithStyle(18,1,"PENALITE DE RETARD ",excel.labelOnYellow);
+        excel.insertWithStyle(6,1,"PRIX TOTAL FOURNITURE HT",excel.labelOnLightYellow);
```
Comment (Rseuret) [inline src/main/java/fr/openent/lystore/export/validOrders/listLycee/ListLycWithPrice.java:258]: refacto

File: src/main/java/fr/openent/lystore/constants/ExportConstants.java [hunk 3]
```diff
@@ -21,4 +21,23 @@ public class ExportConstants {
...
+    public static final String MARKET_CODE_LABEL = "Code marché Région";
+    public static final String REGION_LABEL = "Libellé Région";
+    public static final String DATE_LABEL = "DATE N° RAPPORT";
+    public static final String NUMBER_ORDER_LABEL = "N° de demande";
+    public static final String AMOUNT_LABEL = "Qté";
+    public static final String ROOM_LABEL = "Salle";
+    public static final String STAIR_LABEL = "Étage";
+    public static final String BUILDING_LABEL = "Bâtiment";
+    public static final String ACTIONS_JO = "actionsJO";
+    public static final String NOTIFICATION_COMMENT_REGION = " \n Commentaire Région :";
```
Comment (aymericdevriesere) [inline src/main/java/fr/openent/lystore/constants/ExportConstants.java:38]: Normal le retour à la ligne ?

File: src/main/java/fr/openent/lystore/export/instructions/notificationEquipCP/NotificationLycTab.java [hunk 4]
```diff
@@ -59,95 +55,102 @@ protected void initDatas(Handler<Either<String, Boolean>> handler) {
...
         setStructuresFromDatas(structures);
         datas = sortByUai(datas);
         writeArray();
     }
-//    private void writeArray(Handler<Either<String, Boolean>> handler) {
+
     private void writeArray() {
 
-        log.info("NotificationLYC  writeArray");
+        log.info(ExportConstants.NOTIFICATION_LYC_TAB +" writeArray");
```
Comment (aymericdevriesere) [inline src/main/java/fr/openent/lystore/export/instructions/notificationEquipCP/NotificationLycTab.java:66]: espace après le +

**Pull request merged**